app/domain/audits/serialize_filter_to_query_parameters.rb

```diff
@@ -0,0 +1,21 @@
+module Audits
+  class SerializeFilterToQueryParameters
```

Comment:
Would you say that `Query` parameters belong to the domain layer.
It is a good idea to extract into a serializer the code to create a filter, but I would expect this code to live out of the domain layer. When we talk about query parameters in the domain layer it is likely to think that they are database related?

Any thoughts?

Reply:
Agreed. I've moved into the services directory.